launch/test/launch/test_execute_process.py

```diff
@@ -152,7 +152,7 @@ def generate_launch_description():
         return LaunchDescription([
 
             ExecuteProcess(
-                cmd=[sys.executable, '-c', "print('action')"],
+                cmd=[sys.executable, '-c', "print('action'); false"],
```

Comment:
@lukicdarkoo hmm, `false` is not a valid Python literal.

Reply:
Probably not the best way to make it crash :) I will change it